packages/react-ai/src/components/AIConversation/views/Controls/FieldControl.tsx

```diff
@@ -133,32 +134,35 @@ const InputContainer = withBaseElementProps(View, {
 
 export const FieldControl: FieldControl = () => {
   const { input, setInput } = React.useContext(InputContext);
+  const handleSendMessage = React.useContext(SendMessageContext);
 
   const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
     (e.target as HTMLFormElement).reset();
 
+    const submittedContent: InputContent[] = [];
     if (input?.text) {
       // eslint-disable-next-line @typescript-eslint/no-unused-vars -- TODO send message
```

Comment:
This can be removed now

Reply:
i assume this was pointing to the eslint disabled line, if so i removed that

Reply:
I still see the eslint disabled line?